hypothesis-python/src/hypothesis/internal/conjecture/shrinker.py

```diff
@@ -501,13 +506,14 @@ def s(n):
 
                 total_deleted = self.initial_size - len(self.shrink_target.buffer)
                 calls = self.engine.call_count - self.initial_calls
+                misaligned = self.engine.misaligned_count - self.initial_misaligned
 
                 self.debug(
                     "---------------------\n"
                     "Shrink pass profiling\n"
                     "---------------------\n\n"
                     f"Shrinking made a total of {calls} call{s(calls)} of which "
-                    f"{self.shrinks} shrank. This deleted {total_deleted} bytes out "
+                    f"{self.shrinks} shrank and {misaligned} were misaligned. This deleted {total_deleted} bytes out "
                     f"of {self.initial_size}."
```

Comment:
This message feels pretty confusing to me - maybe we should split it into two points?

- Made {} calls, of which {} were shrinks and {} were misaligned
- Shrank from {} bytes to {} bytes

Reply:
I think this is a bit hard due to space constraints especially for the individual passes per your other comment...I'm not sure how worthwhile it is to make this immediately understandable at the cost of conciseness given it's a core debug message.

is something like this better, or were you imagining something more drastic? (the below is neither worse nor better in my eyes).

```
Shrinking made a total of 178 calls of which 23 shrank and 7 were misaligned.
This deleted 0 bytes out of 21.

Useful passes:

  * minimize_individual_blocks made 67 calls of which 21 shrank and 0 were misaligned. This deleted 0 bytes.
  * dfa_replacement('datetimes()-d66625c3b7') made 7 calls of which 1 shrank and 0 were misaligned. This deleted 0 bytes.

Useless passes:

  * dfa_replacement('emails()-fde8f71142') made 17 calls of which 0 shrank and 0 were misaligned. This deleted 0 bytes.
  * block_program('XX') made 14 calls of which 0 shrank and 0 were misaligned. This deleted 0 bytes.
```


cf

```
Shrinking made a total of 178 calls of which 23 shrank and 7 were misaligned. This deleted 0 bytes out of 21.

Useful passes:

  * minimize_individual_blocks made 67 calls of which 21 shrank and 0 were misaligned, deleting 0 bytes.
  * dfa_replacement('datetimes()-d66625c3b7') made 7 calls of which 1 shrank and 0 were misaligned, deleting 0 bytes.

Useless passes:

  * dfa_replacement('emails()-fde8f71142') made 17 calls of which 0 shrank and 0 were misaligned, deleting 0 bytes.
  * block_program('XX') made 14 calls of which 0 shrank and 0 were misaligned, deleting 0 bytes.
```

Reply:
OK, seeing that example output clears it up for me, go with whatever you prefer.